tensorboard/plugins/debugger/tf_debugger_dashboard/tf-session-runs-view.html

```diff
@@ -157,15 +156,23 @@
       this._clearTable();
       // TODO(cais): Support sorting.
       this._renderHeader();
+      let activeRow;
```

Comment:
This works? I didn't know that we could initialize variables with `let`.

Reply:
Yeah, this is ES6